pytorch_lightning/loggers/comet.py

```diff
@@ -188,7 +188,7 @@ def __init__(
 
     @property
     @rank_zero_experiment
-    def experiment(self):
+    def experiment(self) -> Union[CometExperiment, CometExistingExperiment, CometOfflineExperiment]:
```

Comment:
If this file is fully typed, please remove it from the exclusion list to ensure mypy runs the type checks as part of the CI

Reply:
Unfortunately there are still many typing errors in this file. `CometLogger` was the only logger implementation without typing on the `experiment` property, so I updated it here for consistency. I believe someone else is working on typing the rest of this file. https://github.com/PyTorchLightning/pytorch-lightning/issues/11370